### ISSUE TITLE
sam build for Maven crashing on windows 

### ISSUE THREAD

**Issue by TheSriram** [awslabs/aws-sam-cli#1053]
### Description

Issuing `sam build` for Maven dependency manager with Java runtime on windows can cause the workflow to crash.

### Steps to reproduce

```
sam init --runtime java --dependency-manager maven
sam build
```
### Observed result

```
[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.6.0:exec (default-cli) on project HelloWorld: Command execution failed.: Cannot run program "echo" (in directory "D:\Users\..."): CreateProcess error=2, The system cannot find the file specified -> [Help 1]
```

### Expected result

sam build to work without errors producing build artifacts.

### Temporary Workaround

```
sam build --use-container
```

This runs the maven builder workflow within a lambda like execution environment (lambci docker) and copies artifacts back out from the docker container onto the host machine.

### Additional environment details (Ex: Windows, Mac, Amazon Linux etc)

1. OS: Windows
2. `sam --version`: 0.13.0

`Add --debug flag to command you are running`

**Comment by jfuss** [comment]
This was addressed in `aws-lambda-builders==0.2.1` release, which will go out with the next SAM CLI release.

**Comment by jfuss** [comment]
This was fixed in [v0.14.1](https://github.com/awslabs/aws-sam-cli/releases/tag/v0.14.1).

Closing
